pom.xml

```diff
@@ -11,114 +11,58 @@
     <name>Azure Java Functions</name>
 
     <properties>
+        <java.version>1.8</java.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <azure.functions.maven.plugin.version>1.4.1</azure.functions.maven.plugin.version>
+        <azure.functions.maven.plugin.version>1.3.4</azure.functions.maven.plugin.version>
```

Comment:
The latest maven plugin version is 1.4.1

Reply:
Upgraded